### ISSUE TITLE
Possible max capacity when recording is enabled?

### ISSUE THREAD

**Issue by laurnzdj** [sipwise/rtpengine#828]
Hello everyone
Just a quick question,
Does anyone have any sample load test result or benchmark result on rtpengine with recording option enabled?
Currently on test environment I am using:
OS: Ubuntu 16.04.6 LTS
RTPEngine version: 6.5.4.2
CPU: 8 core
Memory: 8GB
Hard Disk: 500GB

Test flow:
Startrinity01---->Kamailio---->RTPENGINE---->Startrinity02

rtpengine.conf:
```
[rtpengine]
table = 0
interface = internal/10.x.x.x;external/203.x.x.x
listen-ng = 10.x.x.x:33333
timeout = 60
silent-timeout = 120
tos = 184
delete-delay = 15
pidfile = /var/run/ngcp-rtpengine-daemon.pid
port-min = 10000
port-max = 20000
recording-dir = /var/spool/rtpengine
recording-method = proc
```
rtpengine-recording.conf:
```
[rtpengine-recording]
table = 0
output-format = wav
spool-dir = /var/spool/rtpengine 
output-dir = /var/spool/rtpengine_recordings
resample-to = 8000
output-single = 1
```
/etc/sysctl.conf
```
fs.file-max = 100000
net.core.rmem_max=26214400
net.core.rmem_default=26214400
net.core.wmem_max=26214400
net.core.wmem_default=26214400
```

```
root@rtp-sipp01:~# ethtool -g eth0
Ring parameters for eth0:
Pre-set maximums:
RX:		4096
RX Mini:	0
RX Jumbo:	0
TX:		4096
Current hardware settings:
RX:		4096
RX Mini:	0
RX Jumbo:	0
TX:		256

root@rtp-sipp01:~# ethtool -g eth1
Ring parameters for eth1:
Pre-set maximums:
RX:		4096
RX Mini:	0
RX Jumbo:	0
TX:		4096
Current hardware settings:
RX:		4096
RX Mini:	0
RX Jumbo:	0
TX:		256

root@rtp-sipp01:~#
```
With above setup i can reach the following:
When no call recording-option: I can reach 50 cps 1500 calls possibly more
When there is call recording-option: I can only reach 5 cps 100 calls,

Basically I believe I will have lower result when calI recording is on compared when it is not. But still wondering if that is max limit i can reach when call recording is option enabled, that is why i am asking if there was any load test result in the past made with maybe similar setup to compare with or as reference and possible stop looking for possible tuning to increase the capacity.
Thanking you in advance.
